custom_components/homee/switch.py

```diff
@@ -119,6 +119,12 @@ def current_power_w(self):
         """Return the current power usage in W."""
         if self.has_attribute(AttributeType.CURRENT_ENERGY_USE):
             return self.attribute(AttributeType.CURRENT_ENERGY_USE)
+        
```

Comment:
Actually, I think `return None` is missing here.

Reply:
"return none" is still there, but not direclty below "current_power", but below (the new) "total power".
It works for me this way, but that does not mean that the syntax is perfect :)

Reply:
Yes, git confused me here for a second 😄 But every `@property` still should return *something* and with your changes sneaked in `current_power_w` does not return anything anymore if the node does not have the `CURRENT_ENERGY_USE` attribute.

I guess the verbose syntax for the properties should actually be
```python
if self.has_attribute(...):
    return self.attribute(...)
else:
    return None
```

but there was not an explicit `else` because I am lazy and there is already a `return` in the if branch. 
I think it is also stated somewhere in the HA docs that `return None` should be used for "empty" attributes.

Reply:
Ok, I just changed it in the file and tested it on my system. Seems to work :)